src/api/rest/server/server.go

```diff
@@ -106,16 +106,30 @@ func RunServer(port string) {
 		AllowMethods: []string{http.MethodGet, http.MethodPut, http.MethodPost, http.MethodDelete},
 	}))
 
+	// Conditions to prevent abnormal operation due to typos (e.g., ture, falss, etc.)
+	skipBasicAuthOption := os.Getenv("SKIP_BASIC_AUTH") == "true"
```

Comment:
```suggestion
	skipBasicAuthOption := os.Getenv("SKIP_BASIC_AUTH") == "false"
```

기본 옵션이니, env와 동일하게 해두는 것이 일단 좋을 것 같아보이는데 어떠신가요? ^^

이외에는 lgtm입니다!

Reply:
@seokho-son 
"true"인 경우에만 Skip 하도록 만드려는 의도가 있었습니다. 그래서 "true"로 유지하는 것이 좋을 것 같습니다.

예를 들어, "false" 인 경우는 당연히 `false`가 되어 skip하지 않고요. 
"ture", "fals" 등으로 사용자가 잘못 입력하는 경우에도 `false`가 되어 기본적으로 인증을 하게 만들려고 의도하였습니다 ^^

별개로, 사용자가 SKIP을 원하는 경우는, middleware 자체를 활용하지 않는게 바람직한 것으로 판단됩니다. 이 부분은 바로 수정하여 Push하겠습니다.

Reply:
아! 기본값 설정인줄 알고 오해하였습니다. 이슈 없습니다. :)